### PR TITLE
Update AiMod checks

### PR DESCRIPTION
**Pull request by Coppertine**
Set as outdated due to other functions that might have changed to the AIMod that I have not noticed.

### CODE REVIEW

File: wiki/Beatmap_Editor/AiMod/en.md [hunk 1]
```diff
@@ -163,9 +145,8 @@ AiMod typically completes its beatmap analysis in under a minute and subsequentl
 | Title conflicts with {0} diff. | | |
 | Unicode artist conflicts with {0} diff. | | |
 | Unicode title conflicts with {0} diff. | | |
-| This mapset cannot have an Insane diff. without a Hard diff. | | |
-| Filesize of the mapset exceeds 10MB without video. | | |
-| Filesize of the mapset exceeds 24MB with video. | | |
+| This mapset cannot have an Insane diff without a Hard diff. | | |
+| Beatmap exceeds allowed upload size ({0}kb allowed) | The .osz file has found to be larger than the allowed limit set depending on the song. | |
```
Comment (cl8n) [inline wiki/Beatmap_Editor/AiMod/en.md]: probably remove this explanation considering it's super straightforward and none of the others have one rn
```suggestion
| Beatmap exceeds allowed upload size ({0}kb allowed) | | |
```

Comment (Coppertine) [reply wiki/Beatmap_Editor/AiMod/en.md]: Not exactly, the upload that is made is for the zipped file, the upload guidelines looks at the zipped size and not the folder size when unpackaged

Comment (Coppertine) [reply wiki/Beatmap_Editor/AiMod/en.md]: The mapper who would see this error would at first think it's the folder size

Comment (cl8n) [reply wiki/Beatmap_Editor/AiMod/en.md]:
```suggestion
| Beatmap exceeds allowed upload size ({0}kb allowed) | The zipped `.osz` file is too large | |
```

**Pull request merged**